common/catalog/common/haproxy.bom

```diff
@@ -110,7 +110,7 @@ brooklyn.catalog:
               ENTITY_ID: $brooklyn:attributeWhenReady("entity.id")
             command: |
               (
-              flock -o 9
+              flock 9
```

Comment:
Ah, `9` is the file descriptor? See http://stackoverflow.com/questions/8866175/preventing-lock-propagation